test/unit/test_serving.py

```diff
@@ -106,7 +106,7 @@ def test_output_fn_csv(np_array):
     response = serving.default_output_fn(np_array, content_types.CSV)
 
     assert response.get_data(as_text=True) == '1.0,1.0\n1.0,1.0\n'
-    assert response.content_type == content_types.CSV
+    assert content_types.CSV in response.content_type
```

Comment:
Was the content type in the response 'text/csv'? If so, it might be better to check `response.content_type == _content_types.CSV` (_content_types is from sagemaker_containers).

Reply:
content type in response is `text/csv; charset=utf-8`, while sagemaker-containers has `text/csv` https://github.com/aws/sagemaker-containers/blob/master/src/sagemaker_containers/_content_types.py#L15

Reply:
Gotcha, I see Eric also left a comment on the xgboost unit test regarding this.